docs/guides/templates.md

```diff
@@ -24,6 +24,34 @@ npx create-remix --template remix-run/remix/templates/remix-javascript
 
 This is a great place to start if you're just looking to try out Remix for the first time. You can always extend this starting point yourself or migrate to a more advanced template later.
 
+### Official Templates
+
+If you want more control over your server or wish to deploy to a non-node runtime (sch as [Arc][arc], [Deno][deno], or [Cloudflare][cloudflare]), then you can try one of our [official templates][official-templates] from the Remix repository:
+
+```sh
+npx create-remix --template remix-run/remix/templates/arc
+npx create-remix --template remix-run/remix/templates/cloudflare-pages
+npx create-remix --template remix-run/remix/templates/cloudflare-workers
+npx create-remix --template remix-run/remix/templates/deno
+npx create-remix --template remix-run/remix/templates/express
+npx create-remix --template remix-run/remix/templates/fly
+```
```

Comment:
This is the net new info - just exposing the official templates directly in the docs and also providing copy/paste-able commands